tests/testthat/test_search.R

```diff
@@ -1,14 +1,10 @@
 context("search function testing")
 
 test_that("search_stn_number returns a dataframe", {
-  skip_on_cran()
-  skip_on_travis()
-  expect_is(search_stn_number("08HF"), "data.frame")
+  expect_is(search_stn_number("08HF", hydat_path = hy_test_db()), "data.frame")
```

Comment:
I think that this only failed because you didn't have HYDAT downloaded at `hy_dir()`. Now with this approach, these could be tested both on CRAN and Github. Nice!